workflows/prognostic_c48_run/runtime/nudging.py

```diff
@@ -238,23 +238,15 @@ def get_nudging_tendency(
 
 def get_reference_surface_temperatures(state: State, reference: State) -> State:
     """
-    Set the sea surface and surface temperatures in a model state to values in
-    a reference state. Useful for maintaining consistency between a nudged run
-    and reference state.
+    Set the sea surface temperature model state to values in a reference state.
+    Useful for maintaining consistency between a nudged run and reference state.
     """
     state = {
         SST_NAME: _sst_from_reference(
             reference[TSFC_NAME], state[SST_NAME], state[MASK_NAME]
         ),
-        TSFC_NAME: _sst_from_reference(
```

Comment:
Note we do not need to override this variable, so I have removed it.

Reply:
Thanks! I always wondered why this was here, glad to know that it didn't need to be.

Reply:
The nudged-to-obs results got me thinking about this more and I tried another couple tests.  In these tests I included the update to `Sfcprop%tsfc` (`TSFC_NAME` in this context) back in, and tried cases with and without `use_climatological_sst = .true.`.  This time, in both runs, the SST was properly overridden.  

So, what this means is that, similar to the nudged-to-obs case, this was in fact not really particularly broken by VulcanClimateModeling/fv3gfs-fortran#93.  I think what VulcanClimateModeling/fv3gfs-fortran#93 meant was that you needed to override both `tsfc` and `tsfco` to get things to work, when before you only needed to override `tsfco`.  But anyway, I think only having to override one variable is clearer, so perhaps this exercise was useful in the end.

Reply:
I updated [a notebook](https://github.com/VulcanClimateModeling/explore/blob/master/spencerc/2021-03-23-test-sst-update/2021-03-23-verify-sst-setting.ipynb) to show the results of these experiments.  Out of the four options:

1. Set `use_climatological_sst = .true.` and only set the value of the `tsfco` variable in the wrapper/fortran.
2. Set `use_climatological_sst = .false.` and only set the value of the `tsfco` variable in the wrapper/fortran.
3. Set `use_climatological_sst = .true.` and set both `tsfco` and `tsfc` in the wrapper/fortran.
4. Set `use_climatological_sst = .false.` and set both `tsfco` and `tsfc` in the wrapper/fortran.

Only (1), which we haven't ever done anyway, clearly does not work.  (2), (3), and (4) all seem to produce reasonable results for the SST as a diagnostic.  (3) is what we did after VulcanClimateModeling/fv3gfs-fortran#93; (4) is what we did before.  (2) feels the cleanest, but is a departure from what we have done in the past.  In my message above I was sort of hopeful that (2) and (4) would have been equivalent, but that was not true.

After staring at the fortran a while, I really can't say which one of these options is "correct."  I almost wonder now whether we should just fall back to using (3) since it doesn't require making any namelist parameter changes to our configurations.  What do folks think?

Something that also comes up in this notebook are some odd SST differences from the verification data near the coast; I do not know where these are coming from, but I assume they have been in our runs from the beginning.

Reply:
Thanks for checking this all out. In general I think the confusions stems from there being more than one surface temperature variable in the Fortran code. I guess the reason for this is to allow for the "fractional grid" case?

Anyway, when I implemented the SST-setting in Fortran, it seemed safest to me to set both surface temperature variables, and it does seem like that's what we've discovered here. My sense is we should make a decision and move on. I agree option 3 is nice from the perspective of not requiring a new parameter.

Reply:
Yeah, I think the changes that were made to accommodate the fractional grid case make things tricky to follow (the same reason it took a while to settle on VulcanClimateModeling/fv3gfs-fortran#93).